content/posts/2023/reana-0.9.1.md

```diff
@@ -0,0 +1,109 @@
+
+![Launcher examples](/images/reana-0.9.1-launcher-examples.png)
+
+<!-- TODO: add link to docs/blog post here -->
+
+### Automatic closure of inactive sessions
+
+It might happen that users open some interactive sessions and forget to close them when they are not needed anymore.
+To keep resource utilisation under control, REANA 0.9.1 allows to set a maximum time period of inactivity after which inactive sessions will be closed.
+The inactivity period can be customised with the `interactive_sessions.maximum_inactivity_period` Helm value.
+
+To learn more about this feature, please have a look at the related [Configuring interactive sessions](https://docs.reana.io/administration/configuration/configuring-interactive-sessions/#auto-closure-of-inactive-sessions) documentation page.
+
+### Performance and stability improvements
+
+There are many improvements in performance and stability that have landed in this new REANA release:
+
+- The periodic quota updater for disk and CPU quotas is now up to 70% faster.
+- The memory usage of uWSGI and RabbitMQ has been reduced in in environments with a very high number of allowed open files.
+- Static assets of the web interface, such as CSS files and bundled JS sources, are now also served gzip-compressed to lower bandwidth consumption.
+- Monitoring the status of jobs has been improved to avoid some situations in which workflows were not correctly cleaned up in the Kubernetes cluster.
+
+### ... and much more!
+
+Additional features and improvements are present in REANA 0.9.1.
+It is now possible to customise the file size limit for files that can be previewed form the web UI, so that it can be increased or decreased as necessary, for example to allow previewing large ROOT files.
+There are also improvements to how emails are sent, as it is now possible to chose whether to enable SSL or STARTTLS.
+
+### How to upgrade existing REANA 0.9 clusters
+
+<!-- taken and adapted from 0.9.0 blog post -->
+
+If you are a REANA cluster administrator and you would like to upgrade from REANA 0.9.0 to REANA 0.9.1, you can proceed as follows.
+
+First of all, you can optionally define some of the new [Helm values](https://github.com/reanahub/reana/blob/0.9.1/helm/reana/README.md) in your Helm values file:
+
+- `components.reana_ui.file_preview_size_limit` to customise the file size limit for file previews;
+- `login` and `secrets.login` to integrate with an external Keycloak instance;
+- `notifications.email_config.smtp_ssl` and `notifications.email_config.smtp_starttls` to configure how emails are being sent;
+- `ingress.extra` and `ingress.tls.hosts` to further customise Kubernetes Ingresses;
+- `interactive_sessions.maximum_inactivity_period` and `interactive_sessions.cronjob_schedule` to automatically close interactive sessions after some inactivity time;
+- `components.reana_ui.launcher_examples` to choose which examples to show in the main Launch-on-REANA page.
```

Comment:
I have added the list of new Helm values, which is something we were not doing before. What do you think?